scripts/index.js

```diff
@@ -0,0 +1,74 @@
+function getRandomInt(min, max) {
+	min = Math.floor(min);
+	max = Math.ceil(max);
+	return Math.floor(Math.random() * (max - min +1)) + min;
+}
+
+
+let diceOne = {
+	value: 1,
```

Comment:
You can actually assign the initial value of this dice to a random number as well using the `getRandomIntInclusive()` function. That way the dice will always start with a new random number, which will enable for an easier refresh.